plugins/entityreference/selection/OgSelectionHandler.class.php

```diff
@@ -105,7 +105,6 @@ public function buildEntityFieldQuery($match = NULL, $match_operator = 'CONTAINS
 
     $field_mode = $this->instance['field_mode'];
     $user_groups = og_get_groups_by_user(NULL, $group_type);
-    $user_groups = $user_groups ? $user_groups : array();
```

Comment:
this change seems wrong, or atleast we should change `og_get_groups_by_user` to always return `[]`

https://github.com/Gizra/og/blob/81cd92d0651ed97e8d20627a9ebba161673ca6a8/og.module#L3566